examples/sgp4Prop/fsw/src/TestSGP4.c

```diff
@@ -138,7 +138,7 @@ void runVER(VERIN *verins, int cnt)
         fclose(in_file);
     }
 }
-
+/*
 int main(void)
```

Comment:
This change is reverting back to the init commit, looks like I made some small changes in the last pull request but now its back to where it was.